code.json

```
@@ -0,0 +1,74 @@
+[
+  {
+    "name": "ds-pipelines-targets-example-wqp",
+    "organization": "U.S. Geological Survey",
+    "description": "An example targets pipeline for pulling data from the Water Quality Portal (WQP)",
+    "version": "main",
```

If we seek a provisional release, will that be for `main` or specific tags within the repo, e.g. `v0.1.0`?

```suggestion
    "version": "v0.1.0",
```

---

If we only want the release for one version then we can cut back on the duplication. My approach was going to be "try it out"